ravenml/data/write_dataset.py

```diff
@@ -342,6 +271,23 @@ def write_dataset(self, obj_list: list):
         complete_path = dev_path / 'complete'
         self.write_out_complete_set(complete_path, dev_subset)
 
+    def write_out_test_set(self, path, data, associated_files):
+        """Method is helper function for writing out dataset. Writes
+            out test set by copying over associated files to the
+            specified test path. Assumes objlist has 'image_filepath'
+            and 'image_id' as keys.
+
+            If overloaded, there are no expectations.
+
+        Args:
+            path (Path): Path to where data should be written
+            data (list): data that should be written
+            associated_files (list): decides what files are to be copied for the test set
+        """
+        os.mkdir(path)
+        test_image_ids = [(Path(os.path.dirname(obj['image_filepath'])), obj['image_id']) for obj in data]
```

Comment:
The `os.path.dirname` makes it so that it is the path to the imageset directory. However, when I change construct_all to return a dict instead, this logic won't be necessary any longer since the keys should be the image_id tuples in the correct format.